hal/hal.spec

```diff
@@ -102,7 +101,8 @@
 @ stdcall -arch=i386,arm WRITE_PORT_UCHAR(ptr long)
 @ stdcall -arch=i386,arm WRITE_PORT_ULONG(ptr long)
 @ stdcall -arch=i386,arm WRITE_PORT_USHORT(ptr long)
-@ stdcall -arch=x86_64 HalInitializeBios(long ptr)
```

Comment:
Why were these functions previously exported and now not anymore? Did they exist on Win2k3 or not? Or were they only Win7+?

Reply:
( https://www.geoffchappell.com/studies/windows/km/hal/api/x86bios/index.htm )

Reply:
Did you even read that page? Because it explains it in all details.